sources/model/GGRealTimeMontior.m

```diff
@@ -695,7 +695,19 @@ - (void)sendWatchDriverWithDriverUUID:(nonnull NSString *)uuid
     
     
 }
+- (void)sendCustomerSuccessEventWithCustomerAccessToken:(nonnull NSString *)customerAccessToken
+                                  customerId:(nonnull NSString *)customerId
+                                        completionHandler:(nullable SocketResponseBlock)completionHandler {
+    
+    NSMutableDictionary *params = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                   self.developerToken, @"dev_token",
+                                   customerId,@"customer_id",
```

Comment:
does this ignore the case where customer_id/customer_access_token are not provided (i.e i think it shouldn't pass them at all instead of passing null values. anyway just not sure what this does)